openlibrary/plugins/search/code.py

```diff
@@ -239,7 +239,7 @@ def strip_duplicates(seq, init=[]):
 
         # we have somehow gotten some queries for facet tokens with no
         # inverse.  remove these from the list.
-        ft_pairs = filter(lambda (a,b): b, ft_pairs)
+        ft_pairs = filter(lambda a_b: a_b[1], ft_pairs)
```

Comment:
Wow, I wasn't aware of this change in python3, (clearly I spend too much time working with 2.7 :( )  thank you for alerting me to it. More info for anyone else interested: https://www.python.org/dev/peps/pep-3113/

@cclauss I agree with @tfmorris re. breaking this up into PRs that fix only one kind of issue (which I note you've done in #1487 :+1: ) Some change types will take less discussion / review than others and will get merged quicker.

For example, I'm not yet sure how I feel about the lambda signature changes, I guess I'll have to cope though as the PEP has been accepted ;)